aikau/src/main/resources/alfresco/logo/Logo.js

```diff
@@ -115,13 +120,26 @@ define(["dojo/_base/declare",
          this.altText = this.encodeHTML(this.message(this.altText));
          if (this.logoSrc)
          {
-            this.imgNodeStyle = "display: block;";
+            this.imgNodeStyle = "display: inline-block;";
```

Comment:
Doesn't seem to have any impact (having tested 5.0.N with 1.0.40-SNAPSHOT). It's an inline-block inside the root domNode's default block display, so shouldn't really affect the widget's layout on the page.